### PR TITLE
rkt: add different stage1 image support, update README doc

### PR DESCRIPTION
**Pull request by yifan-gu**
Depends on #13455 

/cc @dchen1107 @yujuhong

### CODE REVIEW

File: docs/getting-started-guides/rkt/README.md [hunk 1]
```diff
@@ -38,36 +38,31 @@ We still have [a bunch of work](http://issue.k8s.io/8262) to do to make the expe
...
-  ```console
-  $ sudo systemd-run rkt metadata-service
-  ```
+### Local cluster
 
-  Alternatively, you can use the [rkt-metadata.service](https://github.com/coreos/rkt/blob/master/dist/init/systemd/rkt-metadata.service) and [rkt-metadata.socket](https://github.com/coreos/rkt/blob/master/dist/init/systemd/rkt-metadata.socket) to start the service.
+To use rkt as the container runtime, we need to supply `--container-runtime=rkt` and `--rkt-path=$PATH_TO_RKT_BINARY` to kubelet. Additionally we can provide `--rkt-stage1-image` flag
+as well to select which [stage1 image](https://github.com/coreos/rkt/blob/master/Documentation/running-lkvm-stage1.md) we want to use.
 
+If you are using the [hack/local-up-cluster.sh](../../../hack/local-up-cluster.sh) script to launch the local cluster, then you can edit the environment variable `CONTAINER_RUNTIME`, `RKT_PATH` and `RKT_STAGE1_IAMGE` to
```
Comment (vishh) [inline docs/getting-started-guides/rkt/README.md]: nit: typo in `RKT_STAGE1_IAMGE`

**Comment by vishh** [comment]
LGTM. I will let @yujuhong take a look before adding the LGTM label.

**Comment by yujuhong** [comment]
LGTM except for a few nits. Thanks!

**Comment by yujuhong** [comment]
travis: `Please run hack/update-generated-docs.sh`

**Comment by yifan-gu** [comment]
Rebased

**Comment by wojtek-t** [comment]
@yifan-gu please rebase

**Comment by yifan-gu** [comment]
rebased

**Comment by yifan-gu** [comment]
rebased again

**Comment by yujuhong** [comment]
Thanks for the rebases!

**Comment by yifan-gu** [comment]
ehh, made a typo in `known-flags.txt`,  fixed

**Comment by yujuhong** [comment]
uhh....hope the PR can go in this time...

**Pull request merged**